['/home/giacomo/Development/Programming/python/workspace/Valpa/src/datagen/apps/cavity3d/data/nc144-cpv1-idf12-psBAL_ONE/4e7de57a1a949674b49087051eb9c858d109d5645bafe7ac10fbf5ce34ff370d', '1', '12', '10', 'cavity3d-2014-01-18-04:49-001', 'cavity3d-2014-01-18-04:49-002', 'cavity3d-2014-01-18-04:49-003', 'cavity3d-2014-01-18-04:49-004', 'cavity3d-2014-01-18-04:49-005', 'cavity3d-2014-01-18-04:49-006', 'cavity3d-2014-01-18-04:49-007', 'cavity3d-2014-01-18-04:49-008', 'cavity3d-2014-01-18-04:49-009', 'cavity3d-2014-01-18-04:49-010', '870.62', '881.68', '918.56', '947.58', '967.19', '956.43', '948.53', '916.86', '919.82', '909.98', '/home/giacomo/Development/Programming/python/workspace/Valpa/test/resources/datagen-example/experiments/cavity3d/cfg01']
Traceback (most recent call last):
  File "/usr/lib/python3.2/runpy.py", line 160, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python3.2/runpy.py", line 73, in _run_code
    exec(code, run_globals)
  File "/home/giacomo/Development/Programming/python/workspace/Valpa/src/datagen/runall.py", line 69, in <module>
    run(appName, configVars)
  File "/home/giacomo/Development/Programming/python/workspace/Valpa/src/datagen/runall.py", line 51, in run
    , metricsConfig)
  File "datagen/sargen.py", line 44, in sarAnalyze
    subprocess.call(sargenCall)
  File "/usr/lib/python3.2/subprocess.py", line 471, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python3.2/subprocess.py", line 1404, in wait
    pid, sts = _eintr_retry_call(os.waitpid, self.pid, 0)
  File "/usr/lib/python3.2/subprocess.py", line 456, in _eintr_retry_call
    return func(*args)
KeyboardInterrupt
